spec/lib/api/2.0/users-spec.js

```diff
@@ -0,0 +1,150 @@
+// Copyright 2016, EMC, Inc.
+/* jshint node:true */
+
+'use strict';
+
+describe('Http.Api.Users', function () {
+    var userObj = {
+        username: 'admin',
+        password: 'admin123',
+        role: 'Administrator'
+    };
+
+    var readOnlyObj = {
+        username: 'readonly',
+        password: 'read123',
+        role: 'ReadOnly'
+    };
+
+    var waterline;
+    
+    before('start HTTP server', function () {
+        this.timeout(5000);
+        return helper.startServer([], { authEnabled: true })
+        .then(function() {
+            waterline = helper.injector.get('Services.Waterline');
+            sinon.stub(waterline.localusers, 'create');
+            sinon.stub(waterline.localusers, 'find');
+            sinon.stub(waterline.localusers, 'findOne');
+            sinon.stub(waterline.localusers, 'update');
+            waterline.localusers.findOne.withArgs({username: 'admin'}).resolves({
+                username: userObj.username,
+                comparePassword: function(password) { return password === 'admin123' },
+                role: userObj.role
+            });
+            waterline.localusers.findOne.withArgs({username: 'readonly'}).resolves({
+                username: readOnlyObj.username,
+                comparePassword: function(password) { return password === 'read123' },
+                role: readOnlyObj.role
+            });
+            waterline.localusers.findOne.resolves();
+        });
+    });
+
+    beforeEach(function() {
+        waterline.localusers.create.reset();
+        waterline.localusers.find.reset();
+        waterline.localusers.update.reset();
+        waterline.localusers.findOne.reset();
+    });
+
+    it('should 201 a user post attempt with localexception', function() {
+        waterline.localusers.find.resolves([]);
+        waterline.localusers.create.resolves(userObj);
+        return helper.request().post('/api/2.0/users')
+            .send(userObj)
```

Comment:
the test case process is the same with 'should 401 a user post attempt without auth tokens‘  , IMHO, for use case, any access to /api/2.0/users should be authenticated firstly,  so it would not cause the local exception